engine/flink/util/src/main/scala/pl/touk/nussknacker/engine/flink/util/metrics/MetricUtils.scala

```diff
@@ -91,7 +92,18 @@ class MetricUtils(runtimeContext: RuntimeContext) {
     }
   }
 
-
+  implicit class ExtendedNkGlobalParameters(val parameters: NkGlobalParameters) {
```

Comment:
IMO extension classes are great to handle situation when you have external library and want to add some useful methods, but in our internal classes would be better to introduce just `extractTags(globalParameters)` (it will be more straithforward). What do you think about that?

Reply:
Yes I think you're right, will do